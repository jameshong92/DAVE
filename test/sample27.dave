// test table access
int main() {
    rec r1 = new rec(age: 22, name:"Yang", GPA:3.50); 
    rec r2 = new rec(name:"James", age:23, GPA:3.88);
    rec r3 = new rec(age:24, GPA:3.80, name:"Min Woo");
    rec[] r4 = [r1,r2,r3];
    tbl test = new tbl(r4[0:]);
    print(test);
    fld test1 = access(test, "age");
    print(test1);
	str filename = "testing.txt";
	save(test, filename);
}