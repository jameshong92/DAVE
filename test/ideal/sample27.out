age name    GPA	
22  Yang    3.5	
23  James   3.88	
24  Min Woo 3.8	

age	
22	
23	
24	

